packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts

```diff
@@ -268,10 +270,39 @@ export class AmplifyAuthTransform extends AmplifyCategoryTransform {
     } else if (_.isEmpty(this._cognitoStackProps.triggers)) {
       parameters = Object.assign(parameters, { triggers: JSON.stringify(this._cognitoStackProps.triggers) });
     }
+
+    if (!this.validateCfnParameters(context, oldParameters, parameters)) {
+      process.exit(1);
+    }
+
     //save parameters
     JSONUtilities.writeJson(parametersJSONFilePath, parameters);
   };
 
+  public validateCfnParameters(context: $TSContext, oldParameters: $TSAny, parametersJson: $TSAny) {
```

Comment:
Where are oldParameters sourced from ? Is this from the  #current-cloud-backend ?

Reply:
it's from `amplify/backend/auth/{resourceName}/build/parameters.json`